docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,134 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+Let us to understand how to write tests for ``MyBear`` written in
+``somelanguage``.
+
+.. code::
+
+    import something_necessary
+
+    class MyBear:
+
+        def do_something_cool():
+            does_something_cool
+
+**EXAMPLE 1** using ``verify_local_bear``
+
+.. code::
```

Comment:
did you test this? does this work and it encapsulates with the code tag until line 52? i'm not sure how it works, thats why im asking you